### ISSUE TITLE
PaperUI Inbox should provide Test functionality for discovered devices before adding into aproved things list

### ISSUE THREAD

**Issue by satish-nikam** [eclipse/smarthome#1078]
Discovered devices are shown into PaperUI inbox and user can approve it and add it into things. 

It would be great if PaperUI Inbox page provide "Test Device" functionality right there before approving. This will allow user to provide desired label to the device. e.g. Hue/Lifx binding can discovered all lights in the network, and user should test operation on discovered bulb and decide correct label right there on Inbox Page.

**Comment by kaikreuzer** [comment]
I agree this would be a nice feature, something like an "identify". Many devices won't support it though, so it should only be offered to the user, where it is working. It is not easy to get it nicely into the current architecture, but we can try our best.
